grafana-plugin/src/containers/UserSettings/parts/index.tsx

```diff
@@ -139,9 +133,8 @@ export const TabsContent = observer((props: TabsContentProps) => {
         ) : (
           <PhoneVerification userPk={id} />
         ))}
-      {activeTab === UserSettingsTab.MobileAppVerification && (
-        <MobileAppVerification userPk={id} phone={storeUser.unverified_phone_number || '+'} />
-      )}
+      {/* TODO: we should probably hide this tab when a user (ie. Admin) is viewing the user settings for another user. Would it make sense for an Admin to be able to link their mobile app to another user's profile */}
```

Comment:
general question/comment. should we hide this tab when a user is viewing the settings for another user? I don't think it makes sense to allow user's to link their own mobile app to the profile of another user. Thoughts?